### PR TITLE
*: release 2.1.0

### PR DESCRIPTION
**Pull request by Gouthamve**
This change is [Reviewable](https://reviewable.io/reviews/prometheus/prometheus/3700)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+## 2.1.0 / 2018-1-19
+
+[CHANGE] Remote read `read_recent` option is not false by default.
```
Comment (brian-brazil) [inline CHANGELOG.md]: s/now/not/

This is  BUGFIX/CHANGE, as the said it was false for 2.0.0

Comment (grobie) [inline CHANGELOG.md]: *now I assume. In the docs we already stated in 2.0 that it's false.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+[ENHANCEMENT] Rule evaluation times added to the rules UI.
+[ENHANCEMENT] Added metrics to measure modified time of file SD files.
+[ENHANCEMENT] Kubernetes SD now includes POD UID in discovery metadata.
+[ENHANCEMENT] The Query APIs now return optional stats on query execution times.
+[ENHANCEMENT] The index now no longer has the 4GiB size limit and is also smaller.
+[BUGFIX] Pass the right configuration to each AM when using multiple AM configs.
+[BUGFIX] Fix not-matchers not selecting series with labels unset.
+[BUGFIX] tsdb: Fix occasional panic in headblock.
+[BUGFIX] tsdb: Close files before deletion to fix retention issues on windows and NFS.
+[BUGFIX] tsdb: Cleanup and donot retry failing compactions.
```
Comment (brian-brazil) [inline CHANGELOG.md]: Do not

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
+## 2.1.0 / 2018-1-19
+
+[CHANGE] Remote read `read_recent` option is not false by default.
+[FEATURE] New Service Discovery UI showing labels before and after relabelling.
+[FEATURE] New Admin APIs added to v1 to delete, snapshot and cleanup tombstones.
```
Comment (juliusv) [inline CHANGELOG.md]: "delete, snapshot, and clean up tombstones."

(I was like, "What's a cleanup tombstone?")

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+[ENHANCEMENT] The graph UI autcomplete now includes your previous queries making it easy to debug.
```
Comment (juliusv) [inline CHANGELOG.md]: I'd omit the "making it easy to debug".

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,24 @@
...
+[ENHANCEMENT] Federation is now much faster for large series.
```
Comment (juliusv) [inline CHANGELOG.md]: What's a large series? Do you mean large number of series?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,24 @@
...
+[ENHANCEMENT] Added new metrics to measure rule timings.
+[ENHANCEMENT] Rule evaluation times added to the rules UI.
+[ENHANCEMENT] Added metrics to measure modified time of file SD files.
+[ENHANCEMENT] Kubernetes SD now includes POD UID in discovery metadata.
+[ENHANCEMENT] The Query APIs now return optional stats on query execution times.
+[ENHANCEMENT] The index now no longer has the 4GiB size limit and is also smaller.
+[BUGFIX] Pass the right configuration to each AM when using multiple AM configs.
```
Comment (juliusv) [inline CHANGELOG.md]: I'd spell out AM, not everyone will understand what AM is.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,24 @@
...
+[BUGFIX] Fix not-matchers not selecting series with labels unset.
+[BUGFIX] tsdb: Fix occasional panic in headblock.
```
Comment (juliusv) [inline CHANGELOG.md]: headblock -> head block

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,24 @@
...
+[BUGFIX] tsdb: Close files before deletion to fix retention issues on windows and NFS.
```
Comment (juliusv) [inline CHANGELOG.md]: windows -> Windows

**Comment by zemek** [comment]
Sorry if this was already mentioned but I think you're missing a `*` in front of each line to make them bullet points

**Comment by Gouthamve** [comment]
Thanks! Incorporated all review comments, PTAL. @juliusv @zemek @brian-brazil @grobie

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,24 @@
+## 2.1.0 / 2018-1-19
+
+* [FEATURE] New Service Discovery UI showing labels before and after relabelling.
+* [FEATURE] New Admin APIs added to v1 to delete, snapshot and remove tombstones.
+* [ENHANCEMENT] The graph UI autcomplete now includes your previous queries.
+* [ENHANCEMENT] Federation is now much faster for large number of series.
```
Comment (brian-brazil) [inline CHANGELOG.md]: large numbers

Comment (Gouthamve) [reply CHANGELOG.md]: Done

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**